### ISSUE TITLE
Allow players to switch to "lesser" vision (e.g. disable darkvision)

### ISSUE THREAD

**Issue by ThiefMaster** [foundryvtt/pf2e#14227]
Playing with darkvision is nice. But when you're "just roleplaying" the DM may add very nice lighting effects, but darkvision completely negates those since everything is bright anyway.

It would be nice if there was some way to reduce your own vision.
I thought about creating a custom effect that applies a Sense RE - but apparently it only allows giving you something better, but not something worse.

So it would be nice to have either the ability to make a Sense override whatever there is before (ie basically allowing it to be a debuff instead of a buff), or to have some way as a player to disable your darkvision in the UI (just there's already a "monochrome darkvision" setting, unless that's a core foundry thing).

**Comment by stwlam** [comment]
You can disable rules-based vision on a per-scene basis.

**Comment by ThiefMaster** [comment]
Indeed, that worked fine here. Would still be nice to have an option to downgrade vision at some point ;)
